lib/database/migrations/20231011103554-create-lhc-period.js

```diff
@@ -106,28 +36,43 @@ module.exports = {
             allowNull: true,
         }, { transaction });
 
-        const updateStmt = await prepareRunsUpdateStatement(
-            await createAndGetPeriods({ queryInterface, transaction }),
-            { queryInterface, transaction },
-        );
-        if (updateStmt) {
-            await queryInterface.sequelize.query(updateStmt, { transaction });
-        }
+        const insertPeriodsStmt = `
```

Comment:
Avoid arbitrary shortened names. We usually use `query` (SQL Query), I think statement is too generic